strongbox-commons/src/main/java/org/carlspring/strongbox/io/StreamUtils.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.io;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.carlspring.commons.http.range.ByteRange;
+
+public class StreamUtils
+{
+
+    public static long getLength(ArtifactInputStream is)
```

Comment:
I liked this logic as it was -- in the `ArtifactInputStream` (or it's parent), as it was all contained in one class and you didn't need to know that there was a utility class for this.

What's the benefit of this?